### PR TITLE
Add cancel order by gid

### PR DESCRIPTION
**Pull request by avsek477**
### Description:
Uses the 'oc_multi' event for cancelling multiple orders using group id

### New features:
- Added feature to cancel all orders using group id

### Fixes:
- Missing package "chai" added

### CODE REVIEW

File: lib/ws2/orders/cancel_by_gid.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  emit('exec:order:cancel-by-gid', gid)
+
+  return new Promise((resolve, reject) => {
+    ev.once(`n:oc_multi-req:${gid}:success`, (notification, oc) => {
+      resolve(notification, oc)
+    })
+
+    ev.once(`n:oc_multi-req:${gid}:error`, reject)
+  })
+}
```
Comment (ghost) [inline lib/ws2/orders/cancel_by_gid.js]: memory-safe `watchResponse` from https://github.com/bitfinexcom/bfx-api-node-core/pull/23 should be used after it has been merged

Comment (ghost) [reply lib/ws2/orders/cancel_by_gid.js]: @robertkowalski FYI

Comment (ghost) [reply lib/ws2/orders/cancel_by_gid.js]: @avsek477 https://github.com/bitfinexcom/bfx-api-node-core/pull/23 is merged so you can sync your PR with master and update this code

Comment (avsek477) [reply lib/ws2/orders/cancel_by_gid.js]: rebased to latest master and updated.

**Review by ghost — APPROVED** [review]
lgtm

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**